en/docs/references/enterprise-login.md

```diff
@@ -4,11 +4,13 @@ Choreo's Enterprise Login feature allows your users residing in an external IdP
 
 This guide takes you through the steps you need to follow to configure an enterprise login for your organization in Choreo. 
 
-Let's begin!
+
+## Prerequisites 
+Make sure you have a valid email domain. 
 
 ## Step 1: Create an organization in Choreo
 
-- Sign in to the Choreo Console at <a>https://console.choreo.dev/</a>. using a Google/ GitHub/ Microsoft account.
+- Sign in to the Choreo Console at <a>https://console.choreo.dev/</a> using a Google/ GitHub/ Microsoft account.
```

Comment:
Why is this a bullet point?

Reply:
https://developers.google.com/style/lists

This states not to use lists for a single item.

Reply:
Since this is not only one item, but an item with nested items, I think it's fine to show the hierarchy. Else, the first item will not get the attention it deserves with the two bullet points below it. 
![Screenshot 2022-09-22 at 18 28 12](https://user-images.githubusercontent.com/7173347/191753423-f4075c32-1350-4bca-9681-c8f4831fa5ad.png)

Reply:
![Uploading Screenshot 2022-09-22 at 18.38.38.png…]()
changed to this